### DISCUSSION TITLE
[Bug]: useForm resolver receives different values on 2 subsequent submits on radio input

### DISCUSSION THREAD

**Discussion by stropitek** [react-hook-form/react-hook-form#6297] [Q&A]
### Version Number

7.12.2

### Codesandbox/Expo snack

https://codesandbox.io/s/react-hook-form-null-value-bccy2?file=/src/App.js

### Steps to reproduce

Click on the submit button twice, without selecting any of the radio inputs before

Open the console, you will see that the values received in the resolver callback for the radio input is an empty string the first time, then null the on the second submit

Note that if you change the code slightly so that the resolver returns no errors, then it works as expected.

### Expected behaviour

I would expect the resolver to receive an empty string both times, since this is my default value and I haven't touched the form.

### What browsers are you seeing the problem on?

Edge

### Relevant log output

```shell
values received in resolver callback {radio: ""}
values received in resolver callback {radio: null}
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
This is coming from the `defaultValues`, for consistency, for radio input set defaultValue as `null`.
